stream_alert/sink.py

```diff
@@ -81,6 +84,66 @@ def _get_sns_topic_arn(self):
             topic=topic
         )
 
+    def _chunk_sns_messages(self):
+        """Return a list of base64 encoded SNS messages.
+
+        These are split to ensure each is less than 256kb in size.
+
+        Returns:
+            List of encoded messages that are sent to be published
+            one at a time.
+        """
+
+        chunks = []
+        sns_dict = {'default': []}
+
+        for i in range(len(self.alerts)):
+            added = False
+            sns_json_message = self._get_json_from_dict(sns_dict)
+
+            b64_message_size = self._calculate_b64_size(sns_json_message)
+
+            new_json_message =  ""
+
+            if b64_message_size < _SNS_MAX_SIZE:
+                sns_dict["default"].append(self.alerts[i])
+                new_json_message = self._get_json_from_dict(sns_dict)
+                b64_message_size = self._calculate_b64_size(new_json_message)
+                if b64_message_size >= _SNS_MAX_SIZE:
+                    sns_dict["default"].pop()
+                    encoded_sns_message = base64.b64encode(sns_json_message)
+                    chunks.append(encoded_sns_message)
+                    sns_dict = {'default': [self.alerts[i]]}
+                    added = True
+
+            if not added and (i == len(self.alerts)-1):
+                encoded_sns_message = base64.b64encode(new_json_message)
+                chunks.append(encoded_sns_message)
+
+        return chunks
+
+    def _get_json_from_dict(self, sns_dict):
+
+        def jdefault(obj):
+            """Helper method for marshalling custom objects to JSON"""
+            return obj.__dict__
+
+        json_message = json.dumps(sns_dict, default=jdefault)
+
+        return json_message
+
+    def _calculate_b64_size(self, json_message):
```

Comment:
Code comment says string, arg implies json. Clarify the code comment or change the arg name